privacyideautils/pytoken.template.py

```diff
@@ -2,14 +2,17 @@
 # -*- coding: utf-8 -*-
 
 " HMAC-OTP Software Token "
+from __future__ import print_function
 
+from builtins import str
+from builtins import object
```

Comment:
Hm, what do this imports do? I think the ``builtins`` module doesn't exist under Python 2, but then again, I'm not even sure what this script does in the first place :)

Reply:
crap, missed those... will check.

Reply:
fixed in 2f08a3e

Reply:
fixed in da231fb